src/PeoplePage.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from 'react';
+import { getPeople } from './api';
+import { Person } from './types';
+import { Loader } from './components/Loader';
+import { PersonLink } from './PersonLink';
+
+export const PeoplePage: React.FC<{}> = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    const loadPeople = async () => {
+      try {
+        setPeople(await getPeople());
+      } catch (err) {
+        setError(true);
+      } finally {
+        setIsLoading(false);
+      }
+    };
+
+    loadPeople();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+      {isLoading && <Loader />}
+
+      {error
+        && (
+          <p data-cy="peopleLoadingError" className="has-text-danger">
+            Something went wrong
+          </p>
+        )}
+
+      {(!error && !isLoading)
```

Comment:
If you need to specify more than one condition in the component itself, it is better to put it in a separate variable and name it so that it combines this condition in its name (For example: `isLoadingSuccessful`).